indico/web/client/styles/modules/_category.scss

```diff
@@ -24,6 +24,20 @@
     background: #fff;
     margin-bottom: 10px;
     overflow: hidden;
+
+    .ui.segments > .segment {
+        width: 50%;
+    }
+
+    .chart {
+        width: 96%;
+        height: 350px;
+        padding: 2% 2% 0 2%;
+    }
+
+    td {
```

Comment:
Could we put all this stuff in a separate `category-stats-container`? I think `category-container` is used for all the category pages.